code/src/components/NewTodo.js

```diff
@@ -0,0 +1,71 @@
+import React from "react"
+import { useSelector, useDispatch } from "react-redux"
+import styled from "styled-components"
+
+import todos from "reducers/todos"
+
+const Todo = styled.article`
+  border: 1px solid #dcdcdc;
+  border-radius: 5px;
+  padding: 10px;
+  margin-bottom: 5px;
+  position: relative;
+  background-color: yellow;
+  width: 60%;
+  margin: 0 20% 10px;
+`
+
+const Main = styled.section`
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+`
+
+const DeleteButton = styled.button`
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  border: none;
+  background: none;
+  cursor: pointer;
+`
+
+const NewTodo = () => {
```

Comment:
Again about the naming: maybe it would be clearer what this component is when we read your code if it was called "TodoList" or something like that, since it's not really about a new todo :)